### PR TITLE
UI scrub 1

### PR DESCRIPTION
**Pull request by GraceGardner**
## What does this PR do?
- Removes blue cloud banner
- Adds announcement banner
- Updates Styling : h2/h3 weight and spacing
- Fixes badge positioning for responsive design
- Adds new links
- Updates links for and relocates lang icons 
- Removes worker performance tuning section 
- Fixes font for search drop down on active for contrast 
- Refactors homepage code to clean up and make future updates easier

### CODE REVIEW

File: docs/components/Intro.js [hunk 1]
```diff
@@ -121,200 +122,331 @@ function CliSVG() {
...
-            >
-              join the waitlist
-            </a>
-          </p>
-        </div>
-      </div>
+      <p className="main-subhead">
+        Temporal Platform explanation and usage information.
+      </p>
       <div className="landing-card-section">
```
Comment (djmagee) [inline docs/components/Intro.js:421]: With the Cloud announcement gone, the padding between this div and the subhead above seems excessive to my eye.

**Comment by GraceGardner** [comment]
> Screen Shot 2022-07-07 at 3 00 19 PM
> 
> The spacing for the banner "Temporal Cloud is early access" looks a little smooched.

This banner should be going away with this PR. Did this show up when using the /next slug?

**Review by djmagee — APPROVED** [review]

**Comment by aarohib** [comment]
The header characters seem very tightly packed together:

File: src/css/custom.css [hunk 2]
```diff
@@ -337,6 +337,12 @@ main.max-w-screen-lg {
   font-display: swap;
 }
 
+h2,
+h3 {
+  font-weight: 700;
+  letter-spacing: -2px;
```
Comment (flossypurse) [inline src/css/custom.css:343]: This is too much reduction I think -
Maybe -1 or -.05 or something?

Screen Shot 2022-07-17 at 9 35 23 AM

**Review by flossypurse — COMMENTED** [review]
We need to continue to support this banner until we publish the next release.

Screen Shot 2022-07-17 at 9 36 55 AM

**Comment by flossypurse** [comment]
Closing in favor of the July release PR.

**Pull request closed without merging**